Ginger/Ginger/SourceControl/SourceControlConnDetailsPage.xaml

```diff
@@ -7,23 +7,28 @@
 
+    <ScrollViewer VerticalScrollBarVisibility="Auto">
+        <Grid HorizontalAlignment="Stretch" Background="{StaticResource @Skin1_ColorC}">
+            <StackPanel HorizontalAlignment="Stretch" Margin="5">
+                <Label Content="Source Control Type:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch"  />
+                <TextBox x:Name="SourceControlClassTextBox" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Text="" HorizontalAlignment="Stretch" />
+                <Label Content="Source Control URL:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch" />
+                <TextBox x:Name="SourceControlURLTextBox" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Text="" HorizontalAlignment="Stretch" TextChanged="SourceControlUserDetails_TextChanged" />
+                <Label Content="User ID:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch"/>
+                <TextBox x:Name="SourceControlUserTextBox" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Text="" HorizontalAlignment="Stretch" TextChanged="SourceControlUserDetails_TextChanged"/>
+                <Label  Content="Password:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch"/>
+                <PasswordBox x:Name="SourceControlPassTextBox" Style="{StaticResource @PasswordBoxStyle}" HorizontalAlignment="Stretch"/>
 
-        </StackPanel>
-    </Grid>
+                <CheckBox x:Name="ShowIndicationkForLockedItems"  Content="Show indication for locked items" ToolTip="If checked Ginger will update files icon with Locker icon" Visibility="Collapsed"  HorizontalAlignment="Left" Margin="5,10,0,0"/>
+                <Label Content="Author Name:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch"/>
+                <TextBox x:Name="SourceControlUserAuthorNameTextBox" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Text="" HorizontalAlignment="Stretch" TextChanged="SourceControlUserDetails_TextChanged"/>
+                <Label Content="Author Email:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch"/>
+                <TextBox x:Name="SourceControlAuthorEmailTextBox" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Text="" HorizontalAlignment="Stretch" TextChanged="SourceControlUserDetails_TextChanged"/>
+                <Label x:Name="lblControlConnectionTimeout" Content="Connection Timeout:" Style="{StaticResource @InputFieldLabelStyle}" HorizontalAlignment="Stretch" Visibility="Visible"/>
+                <TextBox x:Name="txtSourceControlConnectionTimeout" Style="{StaticResource @TextBoxStyle}" TextWrapping="Wrap" Visibility="Visible" Text="" HorizontalAlignment="Stretch" TextChanged="SourceControlUserDetails_TextChanged"></TextBox>
```

Comment:
Please check if we need SourceControlUserDetails_TextChanged for timeut text box